package-scripts/datadog-agent/postrm

```diff
@@ -3,8 +3,15 @@ LINUX_DISTRIBUTION=$(grep -Eo "(Debian|Ubuntu|RedHat|CentOS|openSUSE|Amazon)" /e
 
 if [ -f /etc/debian_version ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$LINUX_DISTRIBUTION" == "Ubuntu" ]; then
     set -e
+    
+    # Removing the service form startup (since it's not there anymore)
+    if command -v systemd >/dev/null 2>&1; then
+        systemctl disable datadog-agent
```

Comment:
Indeed there is thanks. The behaviour on Jessie is the expected one, checking on a non systemd distro as soon as my build is complete.